docs-gen/content/rule_set/branches.md

```diff
@@ -27,10 +26,6 @@ well.
 The value ```branch``` specifies that this is a branch entry (as
 opposed to a signal entry). This is the default, in case ```type``` is omitted.
 
-* **```aggregate```** *[optional]*  
-Defines whether or not this branch is an aggregate.
```

Comment:
@gunnarx If we as stated in #256 wants to keep aggregate for use in layers, then should we then not keep it in this file? Possibly with a remark like. "Intended for the future layer concept and not used in VSS core model"

Reply:
I would remove it completely (spec and doc)